ftests/host_test.go

```diff
@@ -6,11 +6,16 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/jingweno/upterm/host/api/swagger/models"
-	"github.com/rs/xid"
```

Comment:
I think we could get rid of the `xid` dependency from go.mod too?

Reply:
Thought I'd commented, clearly hadn't 😂 
`xid` is still being used in the [server/network.go](https://github.com/jingweno/upterm/blob/master/server/network.go#L64) for creating the socket path and in [host/internal/server.go](https://github.com/jingweno/upterm/blob/master/host/internal/server.go#L233) for the TerminalEvent.

These are internal and need to be unique, but we don't have the same worry about predictability since they are not externally controllable/used.

There shouldn't be a significant performance penalty switching these to the same GenerateRandomString, I wouldn't say it is a priority though

Reply:
SGTM. Let's do this in future PR if necessary.